pkgs/os-specific/darwin/apple-source-releases/adv_cmds/package.nix

```diff
@@ -63,6 +77,28 @@ mkAppleDerivation {
       substituteInPlace genwrap/$file \
         --replace-fail '/usr/local' "$out"
     done
+
+    # Set up the locale data
+    unpackFile '${localeSrc}'
+    LOCALE_TOPLEVEL=$PWD/source/usr-share-locale.tproj
+    chmod -R u+w "$LOCALE_TOPLEVEL"
+
+    # bmake expects `Makefile` not `BSDmakefile`.
+    find "$LOCALE_TOPLEVEL" -name Makefile -exec rm {} \; -exec ln -s BSDmakefile {} \;
+
+    # Update `Makefile`s to: get commands from `PATH`, and install to the correct location.
+    # Note: not every `Makefile` has `rsync` or the project name in it.
+    for subproject in colldef mklocale monetdef msgdef numericdef timedef; do
+      substituteInPlace "$LOCALE_TOPLEVEL/$subproject/BSDmakefile" \
+        --replace-warn "../../$subproject.tproj/" "" \
+        --replace-fail /usr/share/locale /share/locale \
+        --replace-fail '-o ''${BINOWN} -g ''${BINGRP}' "" \
+        --replace-warn "rsync -a" "cp -r"
+    done
+
+    # Update `bsdmake` references to `bmake`
+    substituteInPlace "$LOCALE_TOPLEVEL/Makefile" \
+      --replace-fail bsdmake bmake
```

Comment:
I guess we don’t have the exact `bsdmake` Apple expects?

Reply:
We don’t because I deleted it and replaced it with bmake, which is the portable BSD make from NetBSD.